examples/FCCee/flavour/Bc2TauNu/analysis_stage2.py

```diff
@@ -27,6 +27,12 @@
 
 ROOT.gInterpreter.ProcessLine('''
 TMVA::Experimental::RBDT<> bdt("Bc2TauNu_BDT", "/eos/experiment/fcc/ee/analyses/case-studies/flavour/Bc2TauNu/xgb_bdt_vtx.root");
+computeModelBc = TMVA::Experimental::Compute<18, float>(bdt);
+''')
+
+
+ROOT.gInterpreter.ProcessLine('''
+TMVA::Experimental::RBDT<> bdt("Bu2TauNu_BDT", "/afs/cern.ch/work/x/xzuo/public/FCC_files/Bc2TauNu/data/ROOT/xgb_bdt_Bu2TauNu_vtx.root");
 computeModelBis = TMVA::Experimental::Compute<18, float>(bdt);
```

Comment:
could you remove this before I merge, such that the Bc2TauNu stays the same as before?

Reply:
I put it here, but there are other changes in Bc2TauNu that you should revert.

Reply:
Now it is reverted.